spec/spec_helper.rb

```diff
@@ -1,5 +1,25 @@
 require 'simplecov'
 require 'simplecov-console'
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+require_relative './setup_test_database'
+
+ENV['RACK_ENV'] = 'test'
+ENV['ENVIRONMENT'] = 'test'
```

Comment:
Would setting the environment to 'test' for both of these not force the use of the 'chitter_test' database for both automated testing and manual feature testing and not allow for the use of the 'chitter' database?